.github/workflows/code-quality.yml

```
@@ -114,17 +114,17 @@ jobs:
     - name: 🔍 Run MyPy
       run: |
         echo "🔍 Running MyPy type checking..."
-        mypy backend/src/ --ignore-missing-imports --show-error-codes
+        mypy src/ --ignore-missing-imports --show-error-codes
         
     - name: 🔍 Run Pyright
       run: |
         echo "🔍 Running Pyright type checking..."
-        npx pyright backend/src/ || echo "Pyright not available, skipping..."
+        npx pyright src/ || echo "Pyright not available, skipping..."
         
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)